build.cmd

```diff
@@ -332,16 +333,15 @@ REM === Build the CLR VM
 REM ===
 REM =========================================================================================
 
-if %__BuildNative% EQU 1 ( 
+if %__BuildNative% EQU 1 (
+    REM Scope environment changes start {
+    setlocal
```

Comment:
Since you're adding `setlocal` anyway, would an explicit `ENABLEDELAYEDEXPANSION` be a good idea?  I see the script is already using it, so it must be fine to rely on it, I'm just curious.

Reply:
Interestingly, it seems that once you set `ENABLEDELAYEDEXPANSION`, it is set for nested `setlocal` uses.